models/analytics/intermediate/d_shifts_time.sql

```diff
@@ -46,7 +46,7 @@ game_seconds as (
         , case when gs.seconds = shifts.start_seconds_elapsed and shifts.is_period_start is false then true else false end as is_shift_start_not_period_start
         , case when gs.seconds = shifts.start_seconds_elapsed and shifts.is_period_end is true then true else false end as is_shift_end_period_end
 
```

Comment:
same problem as the previous comment with `d_shifts`... good nuff? lol

```
select season_id, count(1) as ct, count(distinct b.shift_id) as shifts, 'gavin' as type
from `nhl-breakouts.dbt_gavin.d_shifts_time` as b
left join `nhl-breakouts.dbt_gavin.d_schedule` as s on s.game_id = b.game_id
group by 1
union all
select season_id, count(1) as ct, count(distinct b.shift_id) as shifts, 'dom' as type
from `nhl-breakouts.dbt_dom.stg_nhl__shifts_time` as b
left join `nhl-breakouts.dbt_dom.d_schedule` as s on s.game_id = b.game_id
group by 1 order by 1
;
```
![image](https://user-images.githubusercontent.com/48701668/204073622-c2f57b39-a57b-4703-8b26-d7b99008f635.png)